.gitignore

```diff
@@ -1,5 +1,6 @@
+.vscode
 *.class
-
+.factorypath
```

Comment:
Could you add these to a local git ignore config instead of to this file?

I acknowledge other tool specific files are in here, I should raise a technical debt issue for removing the others.

Reply:
Can do!  I've removed it from the .gitignore file.